x-pack/test/alerting_api_integration/basic/tests/actions/builtin_action_types/pagerduty.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+import {
+  getExternalServiceSimulatorPath,
+  ExternalServiceSimulator,
+} from '../../../../common/fixtures/plugins/actions';
+
+// eslint-disable-next-line import/no-default-export
+export default function pagerdutyTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('pagerduty action', () => {
+    let pagerdutySimulatorURL: string = '<could not determine kibana url>';
```

Comment:
Nit: I'm thinking we shouldn't even need to get the service simulators set up at all - just pass them an apiUrl (as appropriate) that will at least pass validation.  Since in theory none of the service simulators will be used for these 403 tests.  In the end, not a problem at all, and I guess if someone was looking at these without knowing the passing cases, at least they'd understand there is a concept of these service simulators ...

Reply:
Good catch, I went ahead and fixed that in https://github.com/elastic/kibana/pull/59070/commits/375131e81199fcf0b6ffec373da51c376f815570. Much better!